tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -182,8 +182,6 @@ func podLogs(labelSelector string, container string) {
 	if err != nil {
 		return
 	}
-
-	_, _ = util.Shell("kubectl --namespace %s logs %s -c %s --tail=40 -p", tc.Kube.Namespace, pod, container)
```

Comment:
it's true that the error message is misleading but the fix is not to delete the command as it is needed when the pod 
(the fix would be to capture the output and only log it if the command is successful)
cc @mandarjog